packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -455,6 +459,16 @@ const SystemEntityPage = ({ entity }: { entity: Entity }) => (
       title="Overview"
       element={<SystemOverviewContent entity={entity as SystemEntity} />}
     />
+    <EntityPageLayout.Content
```

Comment:
Good catch. So prior to @Fox32 's addition of tables, the system entities used the default which was to have an overview and docs.  So in my original PR, I had simply extended that so we had overview, docs, and my new diagram tab.  Oliver's change only added the tables to a custom one. But I figured having the docs was still important, so it's sort of a merge issue.  I like the concept of actually having a system entity being able to have docs, in the sense that it might reference an overall architecture or product strategy documentation?  but if the team thinks not at this time, happy to remove it.  thoughts on that?

Reply:
No all good, fair enough, thanks for explaining!